src/apps/testapps/testMaxH3ToChildrenSize.c

```diff
@@ -34,4 +34,16 @@ SUITE(maxH3ToChildrenSize) {
         t_assert(H3_EXPORT(maxH3ToChildrenSize)(parent, 9) == 7 * 7,
                  "got expected size for grandchild res");
     }
+
+    TEST(maxH3ToChildrenSize_largest) {
```

Comment:
Nit: Might be nice to write a quick exhaustive test that took all 122 base cells, and asserted that for each resolution, the sum of `maxH3ToChildrenSize` == `numHexagons` output.

...actually, that won't work, because we pad `maxH3ToChildrenSize` for pentagons. Could we update that function with the closed-form solution for pentagons, and (or v4) change it to `cellToChildrenSize` (no max needed, because now it's exact)?

Reply:
(Could be a follow-up PR, but it would be nice not to over-allocate memory here.)

Reply:
I'm down for that. The only question in the back of my head was whether the `h3ToChildren` function needed the extra workspace or not. If not, then it would be easy to change these to the *exact* size.

Reply:
If my changes [here](https://github.com/uber/h3/pull/409/commits/3c2474f306cedd19d39dee6388d6158cbba9d79f) are correct, it does look like `h3ToChildren` needs the extra workspace, so we can't save on the memory allocation here.

Reply:
Moved to https://github.com/uber/h3/issues/412